app/src/main/java/io/intrepid/thirdpartylibraryexcercise/MainPresenter.java

```diff
@@ -1,5 +1,7 @@
 package io.intrepid.thirdpartylibraryexcercise;
 
+import com.bumptech.glide.Glide;
```

Comment:
unused

Reply:
AS isn't auto-removing unused imports for some reason…

Thanks